service/worker/parentclosepolicy/workflow.go

```diff
@@ -103,38 +103,32 @@ func ProcessorWorkflow(ctx workflow.Context) error {
 
 // ProcessorActivity is activity for processing batch operation
 func ProcessorActivity(ctx context.Context, request Request) error {
+	time.Sleep(2 * time.Minute)
```

Comment:
why sleeps 2 minutes?

Reply:
ah, sorry I forgot to delete the test code. Removed.